packages/tpetra/CMakeLists.txt

```diff
@@ -529,6 +529,29 @@ ELSE () # NOT Tpetra_INST_HIP
     MESSAGE(STATUS "NOTE: Kokkos::HIP is ON (the CMake option Kokkos_ENABLE_HIP is ON), but the corresponding Tpetra Node type is disabled.  If you want to enable instantiation and use of Kokkos::HIP in Tpetra, please also set the CMake option Tpetra_INST_HIP:BOOL=ON.  If you use the Kokkos::HIP version of Tpetra without doing this, you will get link errors!")
   ENDIF ()
 ENDIF () # Tpetra_INST_HIP
+
+# Kokkos::SYCL (Kokkos::Compat::KokkosSYCLWrapperNode)
+GLOBAL_SET(HAVE_TPETRA_INST_SYCL_DEFAULT OFF)
```

Comment:
It would make sense to move this statement further up around line 365 and you might want to add an `ASSERT_DEFINED(Kokkos_ENABLE_SYCL)`

Reply:
See [`a2f73a1` (#9086)](https://github.com/trilinos/Trilinos/pull/9086/commits/a2f73a1a96516642b58fd60078cc4d74f08678c0).